### ISSUE TITLE
Feature: Add gemini

### ISSUE THREAD

**Issue by waleedlatif1** [zaidmukaddam/scira#19]
Should we add support for gemini? Considering it has the highest elo score rn, I believe it should be an option for the models.

**Comment by zaidmukaddam** [comment]
Yeah gemini support in on the way. Gemini doesn't seem to handle array based schema generation very well.

**Comment by waleedlatif1** [comment]
Makes sense.

**Comment by zaidmukaddam** [comment]
going to close this since gemini still hasn't improved a bit lol.
